src/guiKeyChangeMenu.cpp

```diff
@@ -93,13 +95,11 @@ void GUIKeyChangeMenu::removeChildren()
 	const core::list<gui::IGUIElement*> &children = getChildren();
 	core::list<gui::IGUIElement*> children_copy;
 	for (core::list<gui::IGUIElement*>::ConstIterator i = children.begin(); i
-		 != children.end(); i++)
-	{
+		 != children.end(); i++) {
```

Comment:
While you're fixing the code style also put the whole condition on its own line ;)